### PR TITLE
Create Meeting 192.md

### PR DESCRIPTION
**Pull request by Emmanueltech**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ The meetings are independent of any organization. However, Danny Ryan & Tim Beik
 
 | №   | Date                                 | Agenda                                              | Notes                                                                                                                                                                    | Recording                                                                          |
 | --- | ------------------------------------ | --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------- |
+| 190 | July 18, 2024, 14:00-15:30 UTC       | [agenda](https://github.com/ethereum/pm/issues/1098) | [notes](AllCoreDevs-EL-Meetings/Meeting 190.md)\| [Twitter](https://x.com/TimBeiko/status/1813997958939267104)      |[Video](https://youtube.com/live/kL58hvM0E68) |
```
Comment (poojaranjan) [inline README.md:43]:
```suggestion
| 192 | July 18, 2024, 14:00-15:30 UTC       | [agenda](https://github.com/ethereum/pm/issues/1098) | [notes](AllCoreDevs-EL-Meetings/Meeting 192.md)\| [Twitter](https://x.com/TimBeiko/status/1813997958939267104)      |[Video](https://youtube.com/live/kL58hvM0E68) |
| 191 | July 4, 2024, 14:00-15:30 UTC       | [agenda](https://github.com/ethereum/pm/issues/1080) | [notes](AllCoreDevs-EL-Meetings/Meeting 191.md)\| [Twitter] | [Video](https://youtube.com/live/58_bJD_dmm0) |
| 190 | July 4, 2024, 14:00-15:30 UTC       | [agenda](https://github.com/ethereum/pm/issues/1066) | [notes](AllCoreDevs-EL-Meetings/Meeting 190.md)\| [Twitter] | [Video](https://youtube.com/live/8VGf-EE6zNE) |
```

Comment (poojaranjan) [reply README.md:43]: @Emmanueltech please update the README with details added. Thanks.

File: README.md [hunk 2]
```diff
@@ -40,6 +40,9 @@ The meetings are independent of any organization. However, Danny Ryan & Tim Beik
...
 | --- | ------------------------------------ | --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------- |
+| 192 | July 18, 2024, 14:00-15:30 UTC       | [agenda](https://github.com/ethereum/pm/issues/1098) | [notes](AllCoreDevs-EL-Meetings/Meeting 192.md)\| [Twitter](https://x.com/TimBeiko/status/1813997958939267104)      |[Video](https://youtube.com/live/kL58hvM0E68) |
+| 191 | July 4, 2024, 14:00-15:30 UTC       | [agenda](https://github.com/ethereum/pm/issues/1080) | [notes](AllCoreDevs-EL-Meetings/Meeting 191.md)\| [Twitter] | [Video](https://youtube.com/live/58_bJD_dmm0) |
+| 190 | July 4, 2024, 14:00-15:30 UTC       | [agenda](https://github.com/ethereum/pm/issues/1066) | [notes](AllCoreDevs-EL-Meetings/Meeting 190.md)\| [Twitter] | [Video](https://youtube.com/live/8VGf-EE6zNE) |
```
Comment (poojaranjan) [inline README.md:45]:
```suggestion
| 190 | June 20, 2024, 14:00-15:30 UTC       | [agenda](https://github.com/ethereum/pm/issues/1066) | [notes](AllCoreDevs-EL-Meetings/Meeting 190.md)\| [Twitter] | [Video](https://youtube.com/live/8VGf-EE6zNE) |
```

Comment (poojaranjan) [reply README.md:45]: I missed this earlier, apologies. Please update, thanks!

Comment (Emmanueltech) [reply README.md]: > I missed this earlier, apologies. Please update, thanks!

Done

**Comment by poojaranjan** [comment]
@Emmanueltech thanks for documenting this meeting.
README and Notes formatting looks okay. Ready to be merged!

**Comment by Emmanueltech** [comment]
> @Emmanueltech thanks for documenting this meeting. README and Notes formatting looks okay. Ready to be merged!

Thank you Pooja. How do I submit my claim for the bounty?

**Comment by poojaranjan** [comment]
@Emmanueltech Let us take this on the Bounty Issue. Thanks!

**Pull request merged**